pkg/api/operation.go

```diff
@@ -38,3 +38,94 @@ type Display struct {
 	// Friendly name of the operation.
 	Description string `json:"description,omitempty"`
 }
+
+// Common operations defined which can be used within the registration of the APIs
+var OperationResultsRead = Operation{
+	Name: "Microsoft.RedHatOpenShift/locations/operationresults/read",
+	Display: Display{
+		Provider:  "Azure Red Hat OpenShift",
+		Resource:  "locations/operationresults",
+		Operation: "Read operation results",
+	},
+	Origin: "user,system",
+}
+
+var OperationStatusRead = Operation{
+	Name: "Microsoft.RedHatOpenShift/locations/operationsstatus/read",
+	Display: Display{
+		Provider:  "Azure Red Hat OpenShift",
+		Resource:  "locations/operationsstatus",
+		Operation: "Read operations status",
+	},
+	Origin: "user,system",
+}
+
+var OperationRead = Operation{
+	Name: "Microsoft.RedHatOpenShift/operations/read",
+	Display: Display{
+		Provider:  "Azure Red Hat OpenShift",
+		Resource:  "operations",
+		Operation: "Read operations",
+	},
+	Origin: "user,system",
+}
+
+var OperationOpenShiftClusterRead = Operation{
+	Name: "Microsoft.RedHatOpenShift/openShiftClusters/read",
+	Display: Display{
+		Provider:  "Azure Red Hat OpenShift",
+		Resource:  "openShiftClusters",
+		Operation: "Read OpenShift cluster",
+	},
+	Origin: "user,system",
+}
+
+var OperationOpenShiftClusterWrite = Operation{
+	Name: "Microsoft.RedHatOpenShift/openShiftClusters/write",
+	Display: Display{
+		Provider:  "Azure Red Hat OpenShift",
+		Resource:  "openShiftClusters",
+		Operation: "Write OpenShift cluster",
```

Comment:
Might help to specify the full resource type in here - what do you think?
```suggestion
		Operation: "Write Azure Red Hat OpenShift cluster",
```

Reply:
It's specific to our RP, but I'm not sure if we should change the behavior of the output of this operation.  I don't think it would actually do anything  or impact anyone, but i'm not certain to be honest.  I'm going to leave it as is for now.